pkgs/applications/office/gnucash/update.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env nix-shell
+#! nix-shell -i bash -p curl jq nix-prefetch-github
+
+set -euo pipefail
+
+latest_version=$(curl -s https://api.github.com/repos/Gnucash/gnucash/releases/latest | jq -r '.tag_name')
```

Comment:
Fair point, but for now, since we're fetching the release tarball from GitHub, I feel like it makes sense to check the GitHub releases for updates. If we (or upstream) ever switch to another hosting service, we can update things then.